src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsDatasetOptions.java

```diff
@@ -63,37 +63,49 @@ public static void validateOptions(GenomicsDatasetOptions options) {
       Preconditions.checkArgument(0 < options.getBinSize(), "binSize must be greater than zero");
       GenomicsOptions.Methods.validateOptions(options);
     }
-
   }
 
+  @Description("By default, PCA will be run on BRCA1, pass this flag to run on all "
+      + "non X and Y references present in the dataset")
+  boolean isAllReferences();
+  void setAllReferences(boolean allReferences);
+
   @Description("The ID of the Google Genomics dataset this pipeline is working with. "
       + "Defaults to 1000 Genomes.")
   @Default.String("10473108253681171589")
   String getDatasetId();
+  void setDatasetId(String datasetId);
+
+  @Description("The IDs of the Google Genomics call sets this pipeline is working with, comma "
+      + "delimited.Defaults to 1000 Genomes HG00261.")
+  @Default.String("10473108253681171589-0")
+  String getCallSetIds();
```

Comment:
@deflaux let me know if you'd prefer I fork into my own options at this point. Not sure how much we want to jam into this object.